R/toolbar.R

```diff
@@ -7,22 +7,82 @@
 #' @param ... UI elements for the toolbar.
 #' @param align Determines if toolbar should be aligned to the `"right"` or
 #'   `"left"`.
+#' @param gap A CSS length unit defining the gap (i.e., spacing) between
+#'   elements in the toolbar. Defaults to `0` (no gap).
 #' @return Returns a toolbar element.
 #'
 #' @export
 toolbar <- function(
   ...,
-  align = c("right", "left")
+  align = c("right", "left"),
+  gap = NULL
 ) {
   align <- rlang::arg_match(align)
+  gap <- validateCssUnit(gap)
 
   tag <- div(
     class = "bslib-toolbar bslib-gap-spacing",
     "data-align" = align,
+    style = css(gap = gap),
     ...,
     component_dependencies()
   )
 
   tag_require(tag, version = 5, caller = "toolbar()")
   as_fragment(tag)
 }
+
+#' Add toolbar button input
+#'
+#' @description
+#' A button designed to fit well in small places such as toolbars.
+#'
+#' @param id The `input` slot that will be used to access the value.
+#' @param icon An icon to display in the button.
+#' (One of icon or label must be supplied.)
+#' @param label The label to display in the button.
+#' (One of icon or label must be supplied.)
+#' @param tooltip An optional tooltip to display when hovering over the button.
+#' @param disabled If `TRUE`, the button will not be clickable.
+#' Use `updateActionButton()` to dynamically enable/disable the button.
+#' @param border Whether to show a border around the button.
+#' @param ... UI elements for the button.
+#'
+#' @return Returns a button suitable for use in a toolbar.
+#'
```

Comment:
Use this tag to group all of the toolbar component functions together automatically

```suggestion
#'
#' @family toolbar components
#'
```